src/main/resources/public/jsx/DatasetAdder.jsx

```diff
@@ -221,3 +246,11 @@ const LicenceChooser = ({ licenses, currentLicense, onChange }) => {
         </FormGroup>
     )
 }
+
+const CreateCheckbox = ({label, toggleCheckbox}) =>
```

Comment:
The 'key' prop passed in function call is not used (and more generally, this function does not bring a lot)